src/components/ec-checkbox/ec-checkbox.vue

```diff
@@ -30,14 +30,20 @@
           'ec-checkbox__check-icon-wrapper--focused': inputIsFocused,
           'ec-checkbox__check-icon-wrapper--checked': inputModel,
           'ec-checkbox__check-icon-wrapper--checked-and-focused': inputModel && inputIsFocused,
+          'ec-checkbox__check-icon-wrapper--indeterminate': indeterminate,
+          'ec-checkbox__check-icon-wrapper--indeterminate-and-focused': indeterminate && inputIsFocused,
           'ec-checkbox__check-icon-wrapper--error': isInvalid && !inputModel,
           'ec-checkbox__check-icon-wrapper--disabled': disabled,
           'ec-checkbox__check-icon-wrapper--checked-and-disabled': disabled && inputModel,
         }"
         @click="$refs.checkboxInput.click()"
       >
+        <span
+          v-if="indeterminate"
+          class="ec-checkbox__indeterminate-icon"
```

Comment:
I have my thoughts if this should be another icon and swap between icons instead. If no one else mention this you can go ahead and ignore :)

Reply:
The icon for indeterminate is just a white line 10px by 2px. I think it is quite waste of resources trying to extract it into an icon. I don't see the icon to be reusable so it's okay to have it hard coded like this.